### ISSUE TITLE
Custom filters/presets for AFILTER and VFILTER

### ISSUE THREAD

**Issue by dotarmin** [CasparCG/server#680]
It could be really neat to be able to define and write custom AFILTERS and VFILTERS that are accessible in the same way as files in DATA/ etc. It should be possible to apply those custom filters using AMCP.

A custom filter, one complete filter per file (preset is a better name) could be located in a PRESETS folder where each preset is prefixed either with AF_ or VF_.

Best regards,
Armin

**Comment by ronag** [comment]
I think we're moving this into client proxies.

**Comment by dotarmin** [comment]
I think this gets more and more relevant to have some kind of presets (one per file maybe?) that lives on the server based on #1212? Maybe it's not a bad idea anyway but I'm open for suggestions :)

**Comment by premultiply** [comment]
I really would like to have such presets as simple files on the server!

**Comment by dotarmin** [comment]
I can see a lot of use cases but it would be interesting to see if more users in the community has this need.

**Comment by herzogmedia** [comment]
yes, that would be very much appreciated. Even having a default, which is always applied.
